src/bun.js/api/bun/socket.zig

```diff
@@ -1880,6 +1954,104 @@ fn NewSocket(comptime ssl: bool) type {
             return JSValue.jsUndefined();
         }
 
+        pub fn getTLSTicket(
+            this: *This,
+            globalObject: *JSC.JSGlobalObject,
+            _: *JSC.CallFrame,
+        ) callconv(.C) JSValue {
+            if (comptime ssl == false) {
+                return JSValue.jsUndefined();
+            }
+
+            if (this.detached) {
+                return JSValue.jsUndefined();
+            }
+
+            var ssl_ptr: *BoringSSL.SSL = @ptrCast(*BoringSSL.SSL, this.socket.getNativeHandle());
```

Comment:
maybe we should extract this into a helper like `this.socket.ssl()`

Reply:
fix in https://github.com/oven-sh/bun/commit/f2769584f56981d0c362c9224b4ab96d428f2b36#diff-d051905c3e784ffd4b91d0b20f99d7db1a12594b761e3651522925fed42d5bbd